sdk/nodejs/connect.ts

```diff
@@ -68,7 +68,7 @@ export async function connect(
       )
     }
     client = new Client({
-      host: `http://127.0.0.1:${daggerSessionPort}/query`,
+      host: `127.0.0.1:${daggerSessionPort}`,
```

Comment:
@marcosnils : can you add some documentation to the Client for this new format (no scheme prefix, no path suffix)?

Reply:
There's no new format @dolanor! this was a bug that when using the `DAGGER_SESSION_PORT` and `DAGGER_SESSION_TOKEN` variables the URL was constructed incorrectly. There's no need for documentation here.